docs/modules/ROOT/pages/how-tos/release-process.adoc

```diff
@@ -1,7 +1,63 @@
 = Project Syn Tools Release Process
 
 Releases often just require a new SemVer tag with `v`-prefix to be pushed, for example `v1.2.3`.
-The changelog and GitHub release will then be generated automatically.
+
+== Pull request label usage
```

Comment:
I wonder if that section warrants a dedicated page. WDYT?

Reply:
The section does feel a bit out of place here, I would move it to references or explanations and link it here where appropriate.

Reply:
Extracted to #143